edxval/admin.py

```diff
@@ -69,9 +69,19 @@ def get_video_id(self, obj):
 
 
 class VideoTranscriptAdmin(admin.ModelAdmin):
-    list_display = ('video', 'language_code', 'provider', 'file_format')
+    raw_id_fields = ('video',)
+    list_display = ('get_video', 'language_code', 'provider', 'file_format')
+    search_fields = ('id', 'video__edx_video_id', 'language_code')
+
+    def get_video(self, transcript):
+        return transcript.video.edx_video_id if getattr(transcript, 'video', False) else ''
```

Comment:
This if-else seems unnecessary, I think we would always get an associated video with transcript. Isn't this the case ?

Reply:
we have `null=True` constraint on video foreign key at the moment so that Django don't complain to have a default for existing rows. That's why did this.